lib/order.rb

```diff
@@ -0,0 +1,71 @@
+class Order
+
+  require_relative "./menu.rb"
+  require_relative "./menu_displayer.rb"
+
+  attr_reader :total, :cart, :payment_type, :payment
+
+  def initialize(payment_type = "card")
```

Comment:
Snazzy initialised payment type!

Reply:
thanks i have realised in reviewing others work that 
-A. not many others did this
-B. you cant change the payment type back to "card" after changing to "cash" so you'd need to start a fresh order... not much fun on an empty stomach